Tests/Util/PasswordUpdaterTest.php

```diff
@@ -25,6 +27,8 @@ class PasswordUpdaterTest extends TestCase
 
     protected function setUp()
     {
+        BypassFinals::enable();
```

Comment:
rather than hacking a bypass of final classes, I would rather adjust the mocks to rely on the `SelfSaltingEncoderInterface` to detect cases which don't need a salt (which allows mocking).

Another option if we really need to use the actual class is to avoid using a mock.